samples/suit/smp_transfer/sample.yaml

```diff
@@ -59,3 +59,15 @@ tests:
       - CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=2
       - CONFIG_SUIT_DFU_CANDIDATE_PROCESSING_FULL=y
     tags: suit bluetooth
+
+  subsys.suit.smp_transfer.full.extflash.v1:
+    extra_args: FILE_SUFFIX=extflash
+    extra_configs:
+      - CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=1
+    tags: suit
+
+  subsys.suit.smp_transfer.full.extflash.v2:
+    extra_args: FILE_SUFFIX=extflash
+    extra_configs:
+      - CONFIG_SUIT_ENVELOPE_SEQUENCE_NUM=2
```

Comment:
Don't you need to overwrite something to use the templates from `templates/extflash`?

Reply:
The template path is set in `sysbuild_extflash.conf`. The `FILE_SUFFIX` selects it.